templates/data/postgresql/what-is-postgresql.html

```diff
@@ -0,0 +1,636 @@
+        </p>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <div class="p-image-container is-highlighted">
+            <img class="p-image-container__image"
+                 src="https://assets.ubuntu.com/v1/f43e63db-banking.png"
+                 alt="" />
+          </div>
+        </div>
+        <div class="p-equal-height-row__item">
+          <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+          <h5 class="p-heading--5">Banking</h5>
+        </div>
+        <p class="p-equal-height-row__item">
+          PostgreSQL provides the transactional <a href="https://en.wikipedia.org/wiki/ACID">ACID</a> guarantees and the mature replication capabilities that finance systems need. The <a href="https://www.pgaudit.org/">pgAudit</a> extension can complement PostgreSQL with the auditing capabilities that regulated industries require.
+        </p>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <div class="p-image-container is-highlighted">
+            <img class="p-image-container__image"
+                 src="https://assets.ubuntu.com/v1/87f1a153-geospatial-data.png"
+                 alt="" />
+          </div>
+        </div>
+        <div class="p-equal-height-row__item">
+          <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+          <h5 class="p-heading--5">Geospatial data</h5>
+        </div>
+        <p class="p-equal-height-row__item">
+          The <a href="https://postgis.net/">PostGIS</a> extension of PostgreSQL is an excellent choice to support use cases managing spatial data.
+        </p>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <div class="p-image-container is-highlighted">
+            <img class="p-image-container__image"
+                 src="https://assets.ubuntu.com/v1/05d1de66-search.png"
+                 alt="" />
+          </div>
+        </div>
+        <div class="p-equal-height-row__item">
+          <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+          <h5 class="p-heading--5">Search</h5>
+        </div>
+        <p class="p-equal-height-row__item">
+          PostgreSQL offers a variety of index structures (e.g. GIN and GIST) to perform advanced search and similarity operations on your data without the need for duplicating it or compromising on its consistency.
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50-on-large">
```

Comment:
Are we not keeping 50-50 on medium& @mattea-turic

Reply:
I decided not to as it look a little unbalance (lots on the right, very little on the left):
![image](https://github.com/user-attachments/assets/3f486cda-8274-427a-94cb-4da4d4032da4)
But we should get the opinion of @mattea-turic